scout/main.py

```diff
@@ -36,6 +35,7 @@
 
 args = parser.parse_args()
 no_color = args.nocolor
+console = Console(no_color=no_color)
```

Comment:
The good thing with argparse is that it returns `True` if the argument is there and `False` otherwise,
we can use it as the value for `Console`